src/site/stages/build/drupal/graphql/paragraph-fragments/staffProfile.paragraph.graphql.js

```diff
@@ -35,7 +35,7 @@ module.exports = `
                 alt
                 title
                 url
-                derivative(style: _1_1_SQUARE_MEDIUM_THUMBNAIL) {
+                derivative(style: _11SQUAREMEDIUMTHUMBNAIL) {
```

Comment:
Curious why this change was needed?

Reply:
Using `_1_1_SQUARE_MEDIUM_THUMBNAIL` was reported as a GraphQL error and was relying on  `src/site/utilities/stringHelpers.js` to turn it into `_11SQUAREMEDIUMTHUMBNAIL`. Instead of using the `stringHelper`, I just changed the value, because I can't find any context for why it was like that. Asking the CMS folks if they have any idea, https://dsva.slack.com/archives/CDHBKAL9W/p1612907370100500

Reply:
Found it...it's safe to just make this change. The string-helpers that did the replacement before is just leftover complexity from a CMS feature flag, https://github.com/department-of-veterans-affairs/vets-website/commit/7b54b495ccddc229a811231aefba897066aa7e35#diff-78d2ba7081dd26ac2a8a81b3bc6c0c80461f62412580c370dff8564362e33ee4R71.

Reply:
I wrote a ticket to remove this old code, https://github.com/department-of-veterans-affairs/va.gov-team/issues/19722